src/index.ts

```diff
@@ -7,22 +7,21 @@ import * as cors from 'cors';
 import routes from './routes';
 
 //Connects to the Database -> then starts the express
-createConnection()
-	.then(async () => {
-		// Create a new express application instance
-		const app = express();
+createConnection();
 
-		// Call midlewares
-		app.use(cors());
-		app.use(helmet());
-		app.use(bodyParser.json());
+const app = express();
```

Comment:
Hvad er grunden til at fjerne de andre ting fra then efter connection er lavet? 🤔

Reply:
For at kunne exporte det til supertest, ellers var det i et scope hvor den ikke kunne exporte det

Reply:
👍